modules/serverbidBidAdapter.js

```diff
@@ -1,121 +1,75 @@
-      'SMARTSYNC_BASE_URI': 'https://s.zkcdn.net/ss'
-    },
-    'onefiftytwo': {
-      'BASE_URI': 'https://e.serverbid.com/api/v2',
-      'SMARTSYNC_BASE_URI': 'https://s.zkcdn.net/ss'
-    }
-  };
-
-  const SMARTSYNC_CALLBACK = 'serverbidCallBids';
-
-  const sizeMap = [
-    null,
-    '120x90',
-    '120x90',
-    '468x60',
-    '728x90',
-    '300x250',
-    '160x600',
-    '120x600',
-    '300x100',
-    '180x150',
-    '336x280',
-    '240x400',
-    '234x60',
-    '88x31',
-    '120x60',
-    '120x240',
-    '125x125',
-    '220x250',
-    '250x250',
-    '250x90',
-    '0x0',
-    '200x90',
-    '300x50',
-    '320x50',
-    '320x480',
-    '185x185',
-    '620x45',
-    '300x125',
-    '800x250'
-  ];
-
-  sizeMap[77] = '970x90';
-  sizeMap[123] = '970x250';
-  sizeMap[43] = '300x600';
-
-  const bidIds = [];
-
-  baseAdapter.callBids = function(params) {
-    if (params && params.bids &&
-        utils.isArray(params.bids) &&
-        params.bids.length &&
-        CONFIG[params.bidderCode]) {
-      const config = CONFIG[params.bidderCode];
-      config.request = window[params.bidderCode.toUpperCase() + '_CONFIG'];
-      if (!window.SMARTSYNC) {
-        _callBids(config, params);
-      } else {
-        window[SMARTSYNC_CALLBACK] = function() {
-          window[SMARTSYNC_CALLBACK] = function() {};
-          _callBids(config, params);
-        };
-
-        const siteId = params.bids[0].params.siteId;
-        _appendScript(config.SMARTSYNC_BASE_URI + '/' + siteId + '.js');
-
-        const sstimeout = window.SMARTSYNC_TIMEOUT || ((params.timeout || 500) / 2);
-        setTimeout(function() {
-          var cb = window[SMARTSYNC_CALLBACK];
-          window[SMARTSYNC_CALLBACK] = function() {};
-          cb();
-        }, sstimeout);
-      }
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'serverbid';
+
+const CONFIG = {
+  'serverbid': {
+    'BASE_URI': 'https://e.serverbid.com/api/v2'
+  },
+  'connectad': {
+    'BASE_URI': 'https://i.connectad.io/api/v2'
+  },
+  'onefiftytwo': {
+    'BASE_URI': 'https://e.serverbid.com/api/v2'
+  },
+  'insticator': {
+    'BASE_URI': 'https://e.serverbid.com/api/v2'
+  }
+};
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['connectad', 'onefiftytwo', 'insticator'],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.networkId && bid.params.siteId);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+
+  buildRequests: function(validBidRequests) {
+    // Do we need to group by bidder? i.e. to make multiple requests for
```

Comment:
I'm not sure what is meant by grouping by bidder, but multiple requests would be required for different endpoints.

Reply:
@idettman That answers this question, thanks! We aren't going to make a change for this now since it isn't a scenario that has come up yet, but will add in support for that at some point.